projects/angular/src/utils/i18n/common-strings.default.ts

```diff
@@ -68,6 +68,7 @@ export const commonStringsDefault: ClrCommonStrings = {
   datepickerCurrentDecade: 'Current decade',
   datepickerSelectMonthText: 'Select month, the current month is {CALENDAR_MONTH}',
   datepickerSelectYearText: 'Select year, the current year is {CALENDAR_YEAR}',
+  datepickerSelectedLabel: 'Selected',
```

Comment:
I think this should be consistent with `datepickerSelectMonthText` and `datepickerSelectMonthText`.

```typescript
datepickerSelectDayText: 'Select day, the current day is {CALENDER_DAY}`
```